third_party/zlib/CMakeLists.txt

```diff
@@ -27,8 +27,8 @@ endforeach()
 ExternalProject_add(
     zlib
     GIT_REPOSITORY https://github.com/madler/zlib
-    GIT_TAG v1.2.12
+    GIT_TAG v1.2.13
     PREFIX zlib
-    PATCH_COMMAND git checkout . && git apply ${PROJECT_SOURCE_DIR}/build_shared_libs.patch
+    PATCH_COMMAND git checkout . && git apply ${PROJECT_SOURCE_DIR}/build_shared_libs_and_cmake_version.patch
```

Comment:
Can we split that in two PATCH_COMMAND lines? They seem unrelated :thinking:

Reply:
Sure. I just thought putting it into one patch is easier.

Reply:
Ok, as you want. I see patches like commits. It's easier for me to read a commit that does one thing than to try to understand which line corresponds to what when reading the commit.

But if you like one patch better, technically it works :+1:.